### PR TITLE
Update links and other small fixes

### PR DESCRIPTION
**Pull request by big-r81**
- Update links to use https

- Fix typo in master.less/master.css

- Remove Google Analytics Code

- It's 2022 and a smaller dash

- Update whatsnew link to v3.2

- Update IRC links

- Update tm

- Update/fix release notes links to go to correct anker

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -788,38 +788,15 @@ <h2 class="icon icon-more">More &hellip;</h2>
 
   <div id="footer">
 
-    <p>Copyright &copy; 2021 The Apache Software Foundation &mdash; Licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache License 2.0</a><br>
-      Apache CouchDB, CouchDB, and the project logo are <a href="http://www.apache.org/foundation/marks/list/">trademarks</a> of The Apache Software Foundation</p>
+    <p>Copyright &copy; 2022 The Apache Software Foundation &ndash; Licensed under the <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License 2.0</a><br>
```
Comment (janl) [inline index.html]: this is not where ndashes are usually used, why the change?

**Review by janl — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (big-r81) [reply index.html]: Doesn't look that wide.

Comment (janl) [reply index.html]: not a good enough reason :D — I chose the mdash for reasons of typographic conventions.

Comment (big-r81) [reply index.html]: Damn, I'll probably push a new commit ;-)

Comment (big-r81) [reply index.html]: I forgot, we like wider dashes. :D
